src/app/pages/details/details.hbs

```diff
@@ -0,0 +1,172 @@
+<style scoped>
+    @import url(/app/pages/details/details.css);
+    @import url(https://fonts.googleapis.com/css?family=Lato:400,900);
+</style>
+<section class="floating-menu">
+    <div class="title">Get this title</div>
+    <div class="get-this-book"></div>
+    <div class="notes">
+        <sup>*</sup>Our web view is Section 508 Level AA accessible.
+    </div>
+    <div class="cta">
+        <p class="interested-text">
+            Using this book, or interested in adopting it for your
+            course?
+        </p>
+        <a href="/adoptions?{{title}}" class="btn">adopt this book</a>
+        <a class="secondary btn">get interested</a>
+    </div>
+</section>
+<section class="block book-info">
+    <div class="info-row">
+        <img src="https://placeholdit.imgix.net/~text?txtsize=33&txt=Sub-ject&w=200&h=180" />
```

Comment:
Here's a good example of where a global model could be useful.  You are already defining the book's cover URL in subjects.js where you create the model.  This could use that model and then be changed to just `{{cover}}`.